packages/manager/apps/web/client/app/domain/translations/Messages_fr_FR.json

```
@@ -825,5 +825,9 @@
   "domain_tab_options_delete_error": "Une erreur est survenue lors de la résiliation de l'option.",
   "domain_tab_options_delete_cancel": "Annuler",
   "web_domain_zone_activation_success": "Votre activation a été faite avec succès. Votre produit sera activé dans quelques instants.",
-  "web_domain_zone_activation_order_success": "Votre <a href=\"{{t0}}\" target=\"_blank\">bon de commande</a> a été généré avec succès. Votre produit sera activé lorsque vous l'aurez validé."
+  "web_domain_zone_activation_order_success": "Votre <a href=\"{{t0}}\" target=\"_blank\">bon de commande</a> a été généré avec succès. Votre produit sera activé lorsque vous l'aurez validé.",
+  "domain_dashboard_recommended_hosting_title_Pro": "Hébergement de votre site web",
+  "domain_dashboard_recommended_hosting_title_Perso": "Hébergement de votre blog ou votre site web",
+  "domain_dashboard_recommended_hosting_label": "-{{ reduction }} sur votre Hébergement {{ type }} pour le Black Friday.",
```

nitpick:
```suggestion
  "domain_dashboard_recommended_hosting_label": "-{{ discount }} sur votre Hébergement {{ type }} pour le Black Friday.",
```
would be a more accurate translations imo